### ISSUE TITLE
Localization ca.php

### ISSUE THREAD

**Issue by playinteractive** [getopt-php/getopt-php#180]
Here is the translation in Catalan:

```
return [
    'usage-title' => 'Ús: ',
    'usage-command' => 'ordre',
    'usage-options' => 'opcions',
    'usage-operands' => 'operands',
    'operands-title' => "Operands:" . PHP_EOL,
    'options-title' => "Opcions:" . PHP_EOL,
    'commands-title' => "Ordres:" . PHP_EOL,
    'option-unknown' => 'L\'opció \'%s\' és desconeguda',
    'no-more-operands' => 'No s\'esperen més operands - obtingut %s',
    'operand-missing' => 'L\'operand %s és necessari',
    'option-argument-missing' => 'L\'opció \'%s\' ha de tenir un valor',
    'value-invalid' => '%s té un valor invàlid',
];
```

Cheers!

**Comment by tflori** [comment]
thanks for your contribution

**Comment by tflori** [comment]
released with v4.0.3
